chef/cookbooks/ceph/recipes/mon.rb

```diff
@@ -91,7 +91,7 @@
   end
 
   execute "ceph-mon mkfs" do
-    command "ceph-mon --mkfs -i #{node['hostname']} --keyring '#{keyring}'"
+    command "chown ceph:ceph #{keyring} ; ceph-mon --mkfs -i #{node['hostname']} --keyring '#{keyring}' --setuser ceph --setgroup ceph"
```

Comment:
The keyring needs to be owned by the ceph user so the daemons can read it, but (IIRC) because the earlier `ceph-authtool` invocations are being run by chef as root, the keyring ends up with root ownership by default.